src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<blockquote>we would see a value that we did not expect as problem inputs</blockquote><p>"as a problem input", or "among problem inputs" ?</p><p> </p><blockquote>Final step before <code><s>the </s></code>running both problem<code>s</code>, we need <code>to</code> set the value for...</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<blockquote>You will easily notice that the OWE is no longer highlighted in orange, meaning it is now properly connected.</blockquote><p>You will easily notice that the OWE is no longer highlighted in orange, meaning it is now properly connected between <code>compute_fuel_mass</code>and <code>compute_performance</code> . </p>

Reply:
Changed in dfe233f7